test/pytest/service-desktop/test_user_data_backup_restore.py

```diff
@@ -145,22 +146,25 @@ def test_backup_and_restore(harness):
     log.debug("Creating backup")
     assert run_backup(harness), "Failed to initialize backup!"
 
-    harness.connection.watch_port_reboot(20)
-    harness = get_harness_automatic(20)
+    harness.connection.watch_port_reboot(40)
+    harness = get_harness_automatic(40)
+    time.sleep(5)  # Without it harness fails to read from port
```

Comment:
weird

Reply:
I know, but have no idea why this happens. Such "fix" is implemented in other places in the code too.